style.css

```diff
@@ -650,6 +622,136 @@ article h1 {
     outline: none;
     height: 70px;
   }
+
+  .modal {
+    background-color: #e5e5e5;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    overflow: scroll;
+  }
+
+  .hidden-modal {
+    display: none;
+  }
+
+  .modal-box {
+    margin: 5%;
+    width: 70%;
+    height: 150%;
+    background-color: #fff;
+  }
+
+  .modal img {
+    width: 80%;
+    height: 500px;
+    margin-left: 10%;
+    z-index: -7;
+  }
+
+  .modal-box div {
+    height: 50px;
+    width: 100%;
+    padding-right: 40px;
+  }
+
+  .modal .close {
+    font-size: 50px;
+    width: 50px;
+    height: 50px;
+    color: #172b4d;
+    background-color: #e5e5e5;
```

Comment:
- [ ] Dear @danutibe07, As the previous code reviewer mentioned,  Kindly correct the background color from the close button. 👇🏽
### Figma design 
![217996592-93dad669-d3ac-40e5-92b0-67a39a4ce412](https://user-images.githubusercontent.com/69077061/218010274-823ac396-b959-4a03-bd5e-0db5f21aa2d0.png)

### Your design 
![Screenshot 2023-02-10 083025](https://user-images.githubusercontent.com/69077061/218010277-72cd79be-23e8-4ba6-8096-6b48ce54de9e.png)